templates/admin/failedJobs.tpl

```diff
@@ -14,34 +14,49 @@
 		{translate key=$pageTitle}
 	</h1>
```

Comment:
For these simple pages I would consider to actually migrating from smarty to vue. Obviously can be done separately if you prefer. But we will do it at some point anyway

But there are some good benefits, because once you move thing over to Vue file from the smarty (tpl) file it will help you with linting and syntax to make sure that the template is solid.

If you think its good idea to do it as part of this job for simple pages I can provide couple more details what to watch out.